src/EPR.RegulatorService.Facade.API/Controllers/ApplicationController.cs

```diff
@@ -71,15 +74,16 @@ public async Task<IActionResult> GovNotification([FromBody] GovNotificationReque
                 CopyDelegatedUsersFromRequestToModel(request, model);
                 
                 var messageIds = _messagingService.DelegatedPersonRejected(model);
-                _logger.LogInformation($"Emails sent:{String.Join(',', messageIds)}");
+                string logData = string.Format("Emails sent:{0}", String.Join(',', messageIds));
+                _logger.LogInformation("{Message}", logData.Replace('\n', '_'));
                 return Ok(messageIds);
             }
         }
 
-        throw new Exception("Invalid Request");
+        return BadRequest();
```

Comment:
this is change in behaviour please undo this change and just diable the SQ rule and add comment to say this needs fixing in future

Reply:
Done.